erts/etc/darwin/Info.plist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleName</key>
+	<string>Erlang</string>
+	<key>CFBundleDisplayname</key>
+	<string>Erlang</string>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>English</string>
+	<key>CFBundleIdentifier</key>
+	<string>org.erlang.beam</string>
+	<!-- <key>NSUserNotificationAlertStyle</key>
```

Comment:
Can you remove the comments or comment why you still have these left.

Reply:
done